electron/sync.ts

```diff
@@ -251,6 +251,7 @@ export const postFormCloudSubmissions = async (db) => {
         const selectedFile = new Blob([form.xml], { type: 'text/xml' });
         const formData = new FormData();
         formData.append('xml_submission_file', selectedFile, '@/submission.xml');
+        console.log(form.xml);
```

Comment:
Let's not leave console.log's in... I think this should be completely removed.

But if you want to keep it you should probably use `log.debug()` as it works with the winston logger.

Reply:
I can't see the full form object by `log.info(form)` or `log.debug(form)`

Reply:
Hmmm... have you tried JSON.stringifying? It should work fine